.github/workflows/dag-push-production.yaml

```
@@ -0,0 +1,208 @@
+name: Build Packages Prod using dag
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+env:
+  PROJECT: prod-images-c6e5
+  CLUSTER_NAME: tmp-cluster-prod-stage3
+  CLUSTER_ZONE: us-central1-b
+  SERVICE_ACCOUNT: prod-images-ci
+  FQ_SERVICE_ACCOUNT: prod-images-ci@prod-images-c6e5.iam.gserviceaccount.com
+  BUCKET: wolfi-production-registry-source/bootstrap/stage3/
+
+jobs:
+  setup-cluster:
+    name: Setup build cluster
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+      - uses: google-github-actions/auth@v0
+        with:
+          workload_identity_provider: "projects/618116202522/locations/global/workloadIdentityPools/prod-shared-e350/providers/prod-shared-gha"
```

For a follow-up, maybe create a variable with the project id ? There a bunch thorughout
```suggestion
          workload_identity_provider: "projects/${{env.PROJECTID}}/locations/global/workloadIdentityPools/prod-shared-e350/providers/prod-shared-gha"
```